### ISSUE TITLE
0.107.0 Doesn't display correctly

### ISSUE THREAD

**Issue by i00** [thomasloven/lovelace-template-entity-row#20]
State appears under the rest of the left content in 0.107.0:

![image](https://user-images.githubusercontent.com/1435262/77040332-70e93800-6a03-11ea-8964-52216063ee76.png)

```yaml
type: entities
title: My Title
entities:
  - type: custom:template-entity-row
    icon: mdi:lamp
    name: "Name"
    state: "State"
    secondary: "Secondary"
```

**Comment by thomasloven** [comment]
What does your browser console say about "TEMPLATE-ENTITY-ROW"?

**Comment by i00** [comment]
Sorry forgot to mention HA fixed this with 0.107.1

**Comment by nuttytree** [comment]
@thomasloven  I am seeing this issue on HA 0.110.4 (also saw it on 0.109.x and 0.110.3)
Browser console:
